Empty page is added after cover page in order that first page
of the manual itself will be on odd page. This patch
reverts this change, and makes contents centered on page.

@@ -5 +4,0 @@
-\pdfhorigin=1.5cm \pageshift=210mm \advance\pageshift-\hsize \advance\pageshift-2\pdfhorigin
@@ -89 +88 @@
-\pageno=-1 \titletrue\eject\null\vfill\titletrue\eject\pageno=1
+\pageno=0 \titletrue\eject
